pkg/url/url.go

```diff
@@ -59,7 +79,7 @@ func List(client *occlient.Client, componentName string, applicationName string)
 	var urls []URL
 	for _, r := range routes {
 		urls = append(urls, URL{
-			Name:     r.Name,
+			Name:     r.Labels[urlSourceNameAnnotation],
```

Comment:
In `Create` function you assign this value to `Service` and here it is in `Name`

Reply:
Also `Service` is missing here.